cypress/constants/common.ts

```diff
@@ -4,4 +4,7 @@ export const BRIDGE_PASSWORD = 'BRIDGE_KUBEADMIN_PASSWORD';
 
 export const DATA_SERVICES = 'Data Services'; // nav menu category option
 export const DATA_FEDERATION = 'Data Federation'; // nav menu sub category option
-export const STATUS_CARD_TITLE = 'Data Federation service';
+
+export const DATA_FEDERATION_NAMESPACE = 'redhat-data-federation';
```

Comment:
let's keep the constant key as same as the value to remove confusion. REDHAT_DATA_FEDERATION seems to be good

Reply:
we r using same in `package/component` as well... seems okay...